src/gurt/dlog.c

```diff
@@ -106,8 +106,8 @@ static const char        *default_fac0name = "CLOG";
 /* whether we should merge log and stderr */
 static bool               merge_stderr;
 
-#define clog_lock()   pthread_mutex_lock(&clogmux)
-#define clog_unlock() pthread_mutex_unlock(&clogmux)
+#define clog_lock()   (void)pthread_mutex_lock(&clogmux)
+#define clog_unlock() (void)pthread_mutex_unlock(&clogmux)
```

Comment:
I'd forgotten you could do this to indicate that not checking the return was deliberate.

Reply:
Thanks to Jeff and Stack Overflow!

Reply:
I'm aware that you can use it to silence compiler warnings about unchecked returns, it makes sense that coverity behaves the same way.